tests/test_types.py

```diff
@@ -67,6 +67,12 @@ def default(obj: object) -> object:
         0,
         id="bytes",
     ),
+    pytest.param(
+        memoryview(b"a"),
+        b"a",
+        0,
+        id="memoryview",
+    ),
```

Comment:
Any reason to not include also `bytearray` (excluding it from `test_dict_key`) ?

Reply:
Added.